spec/lib/string_spec.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+describe ActiveRecord do
+  describe "with_statement_timeout" do
```

Comment:
How does string munging relate to db transactions? Feels like something has gone off the tracks if modifying a string leads to db issues in the fundamental functionality...

Reply:
woops! Accidentally committed this file